src/lib/main.js

```diff
@@ -179,5 +180,6 @@ instantsearch.createQueryString =
 instantsearch.connectors = connectors;
 instantsearch.widgets = widgets;
 instantsearch.version = version;
+instantsearch.highlight = helpers.highlight;
```

Comment:
It should also be exported from `index.es6.js` to allow users that don't use the full bundled version to use the `highlight` function.

Reply:
same in #3134 then :)

Reply:
Thanks for spotting that!